notebooks/official/feature_store/sdk_feature_store.ipynb

```diff
@@ -0,0 +1,2418 @@
+{
```

Comment:
<p>I'd say don't mention monitoring since SDK did not integrate it, delete this line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>